lib/ansible/modules/network/icx/icx_user.py

```diff
@@ -0,0 +1,394 @@
+        else:
+            if needs_update(want, have, 'configured_password'):
+                if update_password == 'always' or not have:
+                    add(commands, want, '%spassword %s' % ('privilege ' + str(have.get('privilege')) +
+                                                           " " if have.get('privilege') is not None else '', want['configured_password']))
+
+        if needs_update(want, have, 'nopassword'):
+            if want['nopassword']:
+                add(commands, want, 'nopassword')
+
+        if needs_update(want, have, 'access_time'):
+            add(commands, want, 'access-time %s' % want['access_time'])
+
+        if needs_update(want, have, 'expiry_days'):
+            add(commands, want, 'expires %s' % want['expiry_days'])
+
+    return commands
+
+
+def update_objects(want, have):
+    updates = list()
+    for entry in want:
+        item = next((i for i in have if i['name'] == entry['name']), None)
+
+        if all((item is None, entry['state'] == 'present')):
+            updates.append((entry, {}))
+
+        elif all((have == [], entry['state'] == 'absent')):
+            for key, value in iteritems(entry):
+                if key not in ['update_password']:
+                    updates.append((entry, item))
+                    break
+        elif item:
+            for key, value in iteritems(entry):
+                if key not in ['update_password']:
+                    if value is not None and value != item.get(key):
+                        updates.append((entry, item))
+                        break
+    return updates
+
+
+def user_del_cmd(username):
+    return 'no username %s' % username
+
+
+def main():
+    """entry point for module execution
+    """
+    element_spec = dict(
+        name=dict(),
+
+        configured_password=dict(no_log=True),
+        nopassword=dict(type='bool', default=False),
+        update_password=dict(default='always', choices=['on_create', 'always']),
+        privilege=dict(type='str', choices=['0', '4', '5']),
+        access_time=dict(type='str'),
+        state=dict(default='present', choices=['present', 'absent']),
+        check_running_config=dict(default=True, type='bool', fallback=(env_fallback, ['ANSIBLE_CHECK_ICX_RUNNING_CONFIG']))
+    )
+    aggregate_spec = deepcopy(element_spec)
+    aggregate_spec['name'] = dict(required=True)
+
+    remove_default_spec(aggregate_spec)
+
+    argument_spec = dict(
+        aggregate=dict(type='list', elements='dict', options=aggregate_spec, aliases=['users', 'collection']),
+        purge=dict(type='bool', default=False)
+    )
+
+    argument_spec.update(element_spec)
+
+    mutually_exclusive = [('name', 'aggregate')]
+
+    module = AnsibleModule(argument_spec=argument_spec,
+                           mutually_exclusive=mutually_exclusive,
+                           supports_check_mode=True)
+
+    result = {'changed': False}
+    exec_command(module, 'skip')
```

Comment:
Curious to know why do we need this call here?

Reply:
we need to run "skip" to disable page display mode.